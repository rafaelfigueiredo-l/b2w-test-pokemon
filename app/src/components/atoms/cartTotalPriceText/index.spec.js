import CartTotalPriceText from "./";

describe("<CartTotalPriceText />", () => {
  const component = <CartTotalPriceText>Texto</CartTotalPriceText>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly CartTotalPriceText component", () => {
    expect(wrapper.find(".cartTotalPriceText")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly CartTotalPriceText component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".cartTotalPriceText").children()).toHaveLength(1);
  });
});
