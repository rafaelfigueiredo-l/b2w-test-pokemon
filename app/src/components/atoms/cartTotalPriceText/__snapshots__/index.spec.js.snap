// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartTotalPriceText /> should mount correctly CartTotalPriceText component 1`] = `
<styled.span
  className="cartTotalPriceText"
>
  <span
    className="sc-AxjAm cjiwtk cartTotalPriceText"
  >
    Texto
  </span>
</styled.span>
`;
